### ISSUE TITLE
DocumentLayoutException on Linux but not on Windows

### ISSUE THREAD

**Issue by Tvde1** [QuestPDF/QuestPDF#135]
**Describe the bug**
I'm getting a `DocumentLayoutException` on Linux but not on Windows. The element trace is added below.
The culprit seems to be a `column.Text(data)` where `data` has multiple newlines.

On my local windows machine I get a pdf of 10 pages. The element trace ends on a new paragraph on page 3 / 10 (yet the trace shows 4 / 4).

Unfortunately I cannot reproduce it or easily make a MRE. If necessary I'll try.

Any idea how to investigate?

ElementTrace:
```
🔥 Column
---------
Available space: (Width: 14,400.000, Height: 14,400.000)
Requested space: Wrap

    🔥 DebuggingProxy
    -----------------
    Available space: (Width: 14,400.000, Height: 14,400.000)
    Requested space: Wrap

        🔥 Constrained
        --------------
        Available space: (Width: 14,400.000, Height: 14,400.000)
        Requested space: Wrap
        Min Width: 595.4
        Max Width: 595.4
        Min Height: 842
        Max Height: 842

            🔥 Background
            -------------
            Available space: (Width: 595.400, Height: 842.000)
            Requested space: Wrap
            Color: #00000000

                🔥 Padding
                ----------
                Available space: (Width: 595.400, Height: 842.000)
                Requested space: Wrap
                Top: 40
                Right: 50
                Bottom: 40
                Left: 50

                    🔥 DefaultTextStyle
                    -------------------
                    Available space: (Width: 495.400, Height: 762.000)
                    Requested space: Wrap

                        🔥 Decoration
                        -------------
                        Available space: (Width: 495.400, Height: 762.000)
                        Requested space: Wrap

                            Page header 🌟
                            --------------
                            Available space: (Width: 495.400, Height: 762.000)
                            Requested space: FullRender (Width: 495.400, Height: 60.000)

                                Row
                                ---
                                Available space: (Width: 495.400, Height: 762.000)
                                Requested space: FullRender (Width: 495.400, Height: 60.000)

                                    DebuggingProxy
                                    --------------
                                    Available space: (Width: 395.400, Height: 762.000)
                                    Requested space: FullRender (Width: 126.097, Height: 60.000)

                                        Constrained
                                        -----------
                                        Available space: (Width: 395.400, Height: 762.000)
                                        Requested space: FullRender (Width: 126.097, Height: 60.000)
                                        Min Width: -
                                        Max Width: -
                                        Min Height: 60
                                        Max Height: 60

                                            Column
                                            ------
                                            Available space: (Width: 395.400, Height: 60.000)
                                            Requested space: FullRender (Width: 126.097, Height: 25.144)

                                                DebuggingProxy
                                                --------------
                                                Available space: (Width: 395.400, Height: 60.000)
                                                Requested space: FullRender (Width: 126.097, Height: 25.144)

                                                    DefaultTextStyle
                                                    ----------------
                                                    Available space: (Width: 395.400, Height: 60.000)
                                                    Requested space: FullRender (Width: 126.097, Height: 25.144)

                                                        Column
                                                        ------
                                                        Available space: (Width: 395.400, Height: 60.000)
                                                        Requested space: FullRender (Width: 126.097, Height: 25.144)

                                                            DebuggingProxy
                                                            --------------
                                                            Available space: (Width: 395.400, Height: 60.000)
                                                            Requested space: FullRender (Width: 126.097, Height: 25.144)

                                                                TextBlock
                                                                ---------
                                                                Available space: (Width: 395.400, Height: 60.000)
                                                                Requested space: FullRender (Width: 126.097, Height: 25.144)
                                                                Alignment: Left
                                                                Text: [name]

                                    DebuggingProxy
                                    --------------
                                    Available space: (Width: 100.000, Height: 762.000)
                                    Requested space: FullRender (Width: 86.364, Height: 60.000)

                                        Constrained
                                        -----------
                                        Available space: (Width: 100.000, Height: 762.000)
                                        Requested space: FullRender (Width: 86.364, Height: 60.000)
                                        Min Width: -
                                        Max Width: -
                                        Min Height: 60
                                        Max Height: 60

                                            AspectRatio
                                            -----------
                                            Available space: (Width: 100.000, Height: 60.000)
                                            Requested space: FullRender (Width: 86.364, Height: 60.000)
                                            Ratio: 1.439394
                                            Option: FitArea

                                                Image
                                                -----
                                                Available space: (Width: 86.364, Height: 60.000)
                                                Requested space: FullRender (Width: 86.364, Height: 60.000)
                                                Internal Image: SkiaSharp.SKImage

                                    DebuggingProxy
                                    --------------
                                    Available space: (Width: 395.400, Height: 60.000)
                                    Requested space: FullRender (Width: 126.097, Height: 60.000)

                                        Constrained
                                        -----------
                                        Available space: (Width: 395.400, Height: 60.000)
                                        Requested space: FullRender (Width: 126.097, Height: 60.000)
                                        Min Width: -
                                        Max Width: -
                                        Min Height: 60
                                        Max Height: 60

                                    DebuggingProxy
                                    --------------
                                    Available space: (Width: 100.000, Height: 60.000)
                                    Requested space: FullRender (Width: 86.364, Height: 60.000)

                                        Constrained
                                        -----------
                                        Available space: (Width: 100.000, Height: 60.000)
                                        Requested space: FullRender (Width: 86.364, Height: 60.000)
                                        Min Width: -
                                        Max Width: -
                                        Min Height: 60
                                        Max Height: 60

                            Page footer 🌟
                            --------------
                            Available space: (Width: 495.400, Height: 762.000)
                            Requested space: FullRender (Width: 495.400, Height: 37.716)

                                Row
                                ---
                                Available space: (Width: 495.400, Height: 762.000)
                                Requested space: FullRender (Width: 495.400, Height: 37.716)

                                    DebuggingProxy
                                    --------------
                                    Available space: (Width: 198.160, Height: 762.000)
                                    Requested space: FullRender (Width: 105.833, Height: 37.716)

                                        Column
                                        ------
                                        Available space: (Width: 198.160, Height: 762.000)
                                        Requested space: FullRender (Width: 105.833, Height: 37.716)

                                            DebuggingProxy
                                            --------------
                                            Available space: (Width: 198.160, Height: 762.000)
                                            Requested space: FullRender (Width: 88.629, Height: 12.572)

                                                DefaultTextStyle
                                                ----------------
                                                Available space: (Width: 198.160, Height: 762.000)
                                                Requested space: FullRender (Width: 88.629, Height: 12.572)

                                                    Column
                                                    ------
                                                    Available space: (Width: 198.160, Height: 762.000)
                                                    Requested space: FullRender (Width: 88.629, Height: 12.572)

                                                        DebuggingProxy
                                                        --------------
                                                        Available space: (Width: 198.160, Height: 762.000)
                                                        Requested space: FullRender (Width: 88.629, Height: 12.572)

                                                            TextBlock
                                                            ---------
                                                            Available space: (Width: 198.160, Height: 762.000)
                                                            Requested space: FullRender (Width: 88.629, Height: 12.572)
                                                            Alignment: Left
                                                            Text: Exported by:

                                            DebuggingProxy
                                            --------------
                                            Available space: (Width: 198.160, Height: 749.428)
                                            Requested space: FullRender (Width: 105.833, Height: 12.572)

                                                DefaultTextStyle
                                                ----------------
                                                Available space: (Width: 198.160, Height: 749.428)
                                                Requested space: FullRender (Width: 105.833, Height: 12.572)

                                                    Column
                                                    ------
                                                    Available space: (Width: 198.160, Height: 749.428)
                                                    Requested space: FullRender (Width: 105.833, Height: 12.572)

                                                        DebuggingProxy
                                                        --------------
                                                        Available space: (Width: 198.160, Height: 749.428)
                                                        Requested space: FullRender (Width: 105.833, Height: 12.572)

                                                            TextBlock
                                                            ---------
                                                            Available space: (Width: 198.160, Height: 749.428)
                                                            Requested space: FullRender (Width: 105.833, Height: 12.572)
                                                            Alignment: Left
                                                            Text: [name]

                                            DebuggingProxy
                                            --------------
                                            Available space: (Width: 198.160, Height: 736.856)
                                            Requested space: FullRender (Width: 40.764, Height: 12.572)

                                                DefaultTextStyle
                                                ----------------
                                                Available space: (Width: 198.160, Height: 736.856)
                                                Requested space: FullRender (Width: 40.764, Height: 12.572)

                                                    Column
                                                    ------
                                                    Available space: (Width: 198.160, Height: 736.856)
                                                    Requested space: FullRender (Width: 40.764, Height: 12.572)

                                                        DebuggingProxy
                                                        --------------
                                                        Available space: (Width: 198.160, Height: 736.856)
                                                        Requested space: FullRender (Width: 40.764, Height: 12.572)

                                                            TextBlock
                                                            ---------
                                                            Available space: (Width: 198.160, Height: 736.856)
                                                            Requested space: FullRender (Width: 40.764, Height: 12.572)
                                                            Alignment: Left
                                                            Text: [heading]

                                    DebuggingProxy
                                    --------------
                                    Available space: (Width: 99.080, Height: 762.000)
                                    Requested space: FullRender (Width: 20.206, Height: 12.572)

                                        Column
                                        ------
                                        Available space: (Width: 99.080, Height: 762.000)
                                        Requested space: FullRender (Width: 20.206, Height: 12.572)

                                            DebuggingProxy
                                            --------------
                                            Available space: (Width: 99.080, Height: 762.000)
                                            Requested space: FullRender (Width: 20.206, Height: 12.572)

                                                Alignment
                                                ---------
                                                Available space: (Width: 99.080, Height: 762.000)
                                                Requested space: FullRender (Width: 20.206, Height: 12.572)
                                                Vertical: Top
                                                Horizontal: Center

                                                    DefaultTextStyle
                                                    ----------------
                                                    Available space: (Width: 99.080, Height: 762.000)
                                                    Requested space: FullRender (Width: 20.206, Height: 12.572)

                                                        Column
                                                        ------
                                                        Available space: (Width: 99.080, Height: 762.000)
                                                        Requested space: FullRender (Width: 20.206, Height: 12.572)

                                                            DebuggingProxy
                                                            --------------
                                                            Available space: (Width: 99.080, Height: 762.000)
                                                            Requested space: FullRender (Width: 20.206, Height: 12.572)

                                                                TextBlock
                                                                ---------
                                                                Available space: (Width: 99.080, Height: 762.000)
                                                                Requested space: FullRender (Width: 20.206, Height: 12.572)
                                                                Alignment: Center
                                                                Text: 4  /  4

                                    DebuggingProxy
                                    --------------
                                    Available space: (Width: 198.160, Height: 762.000)
                                    Requested space: FullRender (Width: 81.101, Height: 25.144)

                                        Alignment
                                        ---------
                                        Available space: (Width: 198.160, Height: 762.000)
                                        Requested space: FullRender (Width: 81.101, Height: 25.144)
                                        Vertical: Top
                                        Horizontal: Right

                                            Column
                                            ------
                                            Available space: (Width: 198.160, Height: 762.000)
                                            Requested space: FullRender (Width: 81.101, Height: 25.144)

                                                DebuggingProxy
                                                --------------
                                                Available space: (Width: 198.160, Height: 762.000)
                                                Requested space: FullRender (Width: 76.496, Height: 12.572)

                                                    Alignment
                                                    ---------
                                                    Available space: (Width: 198.160, Height: 762.000)
                                                    Requested space: FullRender (Width: 76.496, Height: 12.572)
                                                    Vertical: Top
                                                    Horizontal: Right

                                                        DefaultTextStyle
                                                        ----------------
                                                        Available space: (Width: 198.160, Height: 762.000)
                                                        Requested space: FullRender (Width: 76.496, Height: 12.572)

                                                            Column
                                                            ------
                                                            Available space: (Width: 198.160, Height: 762.000)
                                                            Requested space: FullRender (Width: 76.496, Height: 12.572)

                                                                DebuggingProxy
                                                                --------------
                                                                Available space: (Width: 198.160, Height: 762.000)
                                                                Requested space: FullRender (Width: 76.496, Height: 12.572)

                                                                    TextBlock
                                                                    ---------
                                                                    Available space: (Width: 198.160, Height: 762.000)
                                                                    Requested space: FullRender (Width: 76.496, Height: 12.572)
                                                                    Alignment: Right
                                                                    Text: Export date:

                                                DebuggingProxy
                                                --------------
                                                Available space: (Width: 198.160, Height: 749.428)
                                                Requested space: FullRender (Width: 81.101, Height: 12.572)

                                                    Alignment
                                                    ---------
                                                    Available space: (Width: 198.160, Height: 749.428)
                                                    Requested space: FullRender (Width: 81.101, Height: 12.572)
                                                    Vertical: Top
                                                    Horizontal: Right

                                                        DefaultTextStyle
                                                        ----------------
                                                        Available space: (Width: 198.160, Height: 749.428)
                                                        Requested space: FullRender (Width: 81.101, Height: 12.572)

                                                            Column
                                                            ------
                                                            Available space: (Width: 198.160, Height: 749.428)
                                                            Requested space: FullRender (Width: 81.101, Height: 12.572)

                                                                DebuggingProxy
                                                                --------------
                                                                Available space: (Width: 198.160, Height: 749.428)
                                                                Requested space: FullRender (Width: 81.101, Height: 12.572)

                                                                    TextBlock
                                                                    ---------
                                                                    Available space: (Width: 198.160, Height: 749.428)
                                                                    Requested space: FullRender (Width: 81.101, Height: 12.572)
                                                                    Alignment: Right
                                                                    Text: 08-03-2022 12:00

                                    DebuggingProxy
                                    --------------
                                    Available space: (Width: 198.160, Height: 37.716)
                                    Requested space: FullRender (Width: 105.833, Height: 37.716)

                                        Column
                                        ------
                                        Available space: (Width: 198.160, Height: 37.716)
                                        Requested space: FullRender (Width: 105.833, Height: 37.716)

                                            DebuggingProxy
                                            --------------
                                            Available space: (Width: 198.160, Height: 37.716)
                                            Requested space: FullRender (Width: 88.629, Height: 12.572)

                                                DefaultTextStyle
                                                ----------------
                                                Available space: (Width: 198.160, Height: 37.716)
                                                Requested space: FullRender (Width: 88.629, Height: 12.572)

                                                    Column
                                                    ------
                                                    Available space: (Width: 198.160, Height: 37.716)
                                                    Requested space: FullRender (Width: 88.629, Height: 12.572)

                                                        DebuggingProxy
                                                        --------------
                                                        Available space: (Width: 198.160, Height: 37.716)
                                                        Requested space: FullRender (Width: 88.629, Height: 12.572)

                                                            TextBlock
                                                            ---------
                                                            Available space: (Width: 198.160, Height: 37.716)
                                                            Requested space: FullRender (Width: 88.629, Height: 12.572)
                                                            Alignment: Left
                                                            Text: Exported by:

                                            DebuggingProxy
                                            --------------
                                            Available space: (Width: 198.160, Height: 25.144)
                                            Requested space: FullRender (Width: 105.833, Height: 12.572)

                                                DefaultTextStyle
                                                ----------------
                                                Available space: (Width: 198.160, Height: 25.144)
                                                Requested space: FullRender (Width: 105.833, Height: 12.572)

                                                    Column
                                                    ------
                                                    Available space: (Width: 198.160, Height: 25.144)
                                                    Requested space: FullRender (Width: 105.833, Height: 12.572)

                                                        DebuggingProxy
                                                        --------------
                                                        Available space: (Width: 198.160, Height: 25.144)
                                                        Requested space: FullRender (Width: 105.833, Height: 12.572)

                                                            TextBlock
                                                            ---------
                                                            Available space: (Width: 198.160, Height: 25.144)
                                                            Requested space: FullRender (Width: 105.833, Height: 12.572)
                                                            Alignment: Left
                                                            Text: [name]

                                            DebuggingProxy
                                            --------------
                                            Available space: (Width: 198.160, Height: 12.572)
                                            Requested space: FullRender (Width: 40.764, Height: 12.572)

                                                DefaultTextStyle
                                                ----------------
                                                Available space: (Width: 198.160, Height: 12.572)
                                                Requested space: FullRender (Width: 40.764, Height: 12.572)

                                                    Column
                                                    ------
                                                    Available space: (Width: 198.160, Height: 12.572)
                                                    Requested space: FullRender (Width: 40.764, Height: 12.572)

                                                        DebuggingProxy
                                                        --------------
                                                        Available space: (Width: 198.160, Height: 12.572)
                                                        Requested space: FullRender (Width: 40.764, Height: 12.572)

                                                            TextBlock
                                                            ---------
                                                            Available space: (Width: 198.160, Height: 12.572)
                                                            Requested space: FullRender (Width: 40.764, Height: 12.572)
                                                            Alignment: Left
                                                            Text: [heading]

                                    DebuggingProxy
                                    --------------
                                    Available space: (Width: 99.080, Height: 37.716)
                                    Requested space: FullRender (Width: 20.206, Height: 12.572)

                                        Column
                                        ------
                                        Available space: (Width: 99.080, Height: 37.716)
                                        Requested space: FullRender (Width: 20.206, Height: 12.572)

                                            DebuggingProxy
                                            --------------
                                            Available space: (Width: 99.080, Height: 37.716)
                                            Requested space: FullRender (Width: 20.206, Height: 12.572)

                                                Alignment
                                                ---------
                                                Available space: (Width: 99.080, Height: 37.716)
                                                Requested space: FullRender (Width: 20.206, Height: 12.572)
                                                Vertical: Top
                                                Horizontal: Center

                                                    DefaultTextStyle
                                                    ----------------
                                                    Available space: (Width: 99.080, Height: 37.716)
                                                    Requested space: FullRender (Width: 20.206, Height: 12.572)

                                                        Column
                                                        ------
                                                        Available space: (Width: 99.080, Height: 37.716)
                                                        Requested space: FullRender (Width: 20.206, Height: 12.572)

                                                            DebuggingProxy
                                                            --------------
                                                            Available space: (Width: 99.080, Height: 37.716)
                                                            Requested space: FullRender (Width: 20.206, Height: 12.572)

                                                                TextBlock
                                                                ---------
                                                                Available space: (Width: 99.080, Height: 37.716)
                                                                Requested space: FullRender (Width: 20.206, Height: 12.572)
                                                                Alignment: Center
                                                                Text: 4  /  4

                                    DebuggingProxy
                                    --------------
                                    Available space: (Width: 198.160, Height: 37.716)
                                    Requested space: FullRender (Width: 81.101, Height: 25.144)

                                        Alignment
                                        ---------
                                        Available space: (Width: 198.160, Height: 37.716)
                                        Requested space: FullRender (Width: 81.101, Height: 25.144)
                                        Vertical: Top
                                        Horizontal: Right

                                            Column
                                            ------
                                            Available space: (Width: 198.160, Height: 37.716)
                                            Requested space: FullRender (Width: 81.101, Height: 25.144)

                                                DebuggingProxy
                                                --------------
                                                Available space: (Width: 198.160, Height: 37.716)
                                                Requested space: FullRender (Width: 76.496, Height: 12.572)

                                                    Alignment
                                                    ---------
                                                    Available space: (Width: 198.160, Height: 37.716)
                                                    Requested space: FullRender (Width: 76.496, Height: 12.572)
                                                    Vertical: Top
                                                    Horizontal: Right

                                                        DefaultTextStyle
                                                        ----------------
                                                        Available space: (Width: 198.160, Height: 37.716)
                                                        Requested space: FullRender (Width: 76.496, Height: 12.572)

                                                            Column
                                                            ------
                                                            Available space: (Width: 198.160, Height: 37.716)
                                                            Requested space: FullRender (Width: 76.496, Height: 12.572)

                                                                DebuggingProxy
                                                                --------------
                                                                Available space: (Width: 198.160, Height: 37.716)
                                                                Requested space: FullRender (Width: 76.496, Height: 12.572)

                                                                    TextBlock
                                                                    ---------
                                                                    Available space: (Width: 198.160, Height: 37.716)
                                                                    Requested space: FullRender (Width: 76.496, Height: 12.572)
                                                                    Alignment: Right
                                                                    Text: Export date:

                                                DebuggingProxy
                                                --------------
                                                Available space: (Width: 198.160, Height: 25.144)
                                                Requested space: FullRender (Width: 81.101, Height: 12.572)

                                                    Alignment
                                                    ---------
                                                    Available space: (Width: 198.160, Height: 25.144)
                                                    Requested space: FullRender (Width: 81.101, Height: 12.572)
                                                    Vertical: Top
                                                    Horizontal: Right

                                                        DefaultTextStyle
                                                        ----------------
                                                        Available space: (Width: 198.160, Height: 25.144)
                                                        Requested space: FullRender (Width: 81.101, Height: 12.572)

                                                            Column
                                                            ------
                                                            Available space: (Width: 198.160, Height: 25.144)
                                                            Requested space: FullRender (Width: 81.101, Height: 12.572)

                                                                DebuggingProxy
                                                                --------------
                                                                Available space: (Width: 198.160, Height: 25.144)
                                                                Requested space: FullRender (Width: 81.101, Height: 12.572)

                                                                    TextBlock
                                                                    ---------
                                                                    Available space: (Width: 198.160, Height: 25.144)
                                                                    Requested space: FullRender (Width: 81.101, Height: 12.572)
                                                                    Alignment: Right
                                                                    Text: 08-03-2022 12:00

                            🔥 Extend
                            ---------
                            Available space: (Width: 495.400, Height: 664.284)
                            Requested space: Wrap
                            Extend Vertical: False
                            Extend Horizontal: True

                                🔥 Extend
                                ---------
                                Available space: (Width: 495.400, Height: 664.284)
                                Requested space: Wrap
                                Extend Vertical: True
                                Extend Horizontal: False

                                    🔥 Page content 🌟
                                    ------------------
                                    Available space: (Width: 495.400, Height: 664.284)
                                    Requested space: Wrap

                                        🔥 Column
                                        ---------
                                        Available space: (Width: 495.400, Height: 664.284)
                                        Requested space: Wrap

                                            🔥 DebuggingProxy
                                            -----------------
                                            Available space: (Width: 495.400, Height: 664.284)
                                            Requested space: Wrap

                                                🔥 Column
                                                ---------
                                                Available space: (Width: 495.400, Height: 664.284)
                                                Requested space: Wrap

                                                    🔥 DebuggingProxy
                                                    -----------------
                                                    Available space: (Width: 495.400, Height: 664.284)
                                                    Requested space: Wrap

                                                        🔥 Border
                                                        ---------
                                                        Available space: (Width: 495.400, Height: 664.284)
                                                        Requested space: Wrap
                                                        Color: #eeeeee
                                                        Top: 3
                                                        Right: 3
                                                        Bottom: 3
                                                        Left: 3

                                                            🔥 Padding
                                                            ----------
                                                            Available space: (Width: 495.400, Height: 664.284)
                                                            Requested space: Wrap
                                                            Top: 10
                                                            Right: 10
                                                            Bottom: 10
                                                            Left: 10

                                                                🔥 Column
                                                                ---------
                                                                Available space: (Width: 475.400, Height: 644.284)
                                                                Requested space: Wrap

                                                                    🔥 DebuggingProxy
                                                                    -----------------
                                                                    Available space: (Width: 475.400, Height: 644.284)
                                                                    Requested space: Wrap

                                                                        🔥 Column
                                                                        ---------
                                                                        Available space: (Width: 475.400, Height: 644.284)
                                                                        Requested space: Wrap

                                                                            🔥 DebuggingProxy
                                                                            -----------------
                                                                            Available space: (Width: 475.400, Height: 644.284)
                                                                            Requested space: Wrap

                                                                                🔥 DefaultTextStyle
                                                                                -------------------
                                                                                Available space: (Width: 475.400, Height: 644.284)
                                                                                Requested space: Wrap

                                                                                    🔥 Column
                                                                                    ---------
                                                                                    Available space: (Width: 475.400, Height: 644.284)
                                                                                    Requested space: Wrap

                                                                                        🔥 DebuggingProxy
                                                                                        -----------------
                                                                                        Available space: (Width: 475.400, Height: 644.284)
                                                                                        Requested space: Wrap

                                                                                            🔥 TextBlock
                                                                                            ------------
                                                                                            Available space: (Width: 475.400, Height: 644.284)
                                                                                            Requested space: Wrap
                                                                                            Alignment: Left
                                                                                            Text: [text omitted]
```

**Comment by MarcinZiabek** [comment]
Hello 😁

1. Please share the code that generates the PDF document (especially text content).
2. Do you use the default library font or do you specify a custom font name?

**Comment by Tvde1** [comment]
I've done quite a bit of testing, and it seems to occur when text is inserted ending with a period and a space, where the space is wrapped to the next line. I'm trying to make an example project to isolate that that is the problem.

**Comment by Tvde1** [comment]
On my windows machine, this example throws the document exception. I specifically crafted the string length. Adding one character wraps the sentence to a 2nd line and removing one character keeps the sentence on the same line. The space after the period might not be necessary, haven't tested.
```
public override void Compose(IDocumentContainer container)
{
    container.Page(page =>
    {
        page.Margin(50);

        page.Size(PageSizes.A4);

        page.Content()
            .Text(
                "This is a specially crafted sentence with a specially chosen length for demonstration of the bug that occurs ;;;;;. ",
                TextStyle.Default.Size(11));
    });
}
```

**Comment by Tvde1** [comment]
And for the default text size, the string `"This is a specially crafted sentence with a specially chosen length for demonstration of the bug that occurs ;;;;;. "` works on my windows machine

**Comment by MarcinZiabek** [comment]
Thank you for finding this issue and performing great investigation! Your example code really has helped me finding the root cause. Fortunatelly, it is a very specific corner case and it should not affect many people. 

I just released QuestPDF 2022.2.6. Can you please try it and confirm that the issue is fixed?

**Comment by Tvde1** [comment]
Works like a charm!
Thanks for the swift response & fix! I wish all open source libraries were like that :^)

This was fun to debug :) Especially since the rendering results in a different pdf on linux than on windows.
